types/npmlog/index.d.ts

```diff
@@ -15,17 +15,29 @@ declare namespace npmlog {
         heading: string;
         stream: any; // Defaults to process.stderr
 
-        log(level: LogLevels | string, prefix: string, message: string, ...args: any[]): void;
+        /**
+         * Creates a log message
+         * @param level
+         * @param prefix
+         * @param message message of the lopg which will be formatted using utils.format()
+         * @param args additional arguments
```

Comment:
Maybe specify that those are additional arguments that are passed to `utils.format`, too?

Reply:
Done